### DISCUSSION TITLE
有没有砖家知道mpv怎么看片源的色深是多为位的呀

### DISCUSSION THREAD

**Discussion by OrangeYoon** [hooke007/mpv_PlayKit#464] [mpv 通用]
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/4724a6b5-5c81-406e-abfd-d503abb9a5c8)
我这边不知道哪个参数是颜色位数，应该不是8就是10

**Comment by hooke007** [comment]
![image](https://github.com/hooke007/MPV_lazy/assets/41094733/65875357-0881-4029-ba84-c1920a047298)
8位
yuv420p8通常省略不写末位的8

**Reply by OrangeYoon** [reply]
还有我的显示器说是支持8抖10的lcd面板， dither-depth需要设置什么呢，不知道auto是什么情况，用的是gpu-next

**Reply by hooke007** [reply]
d3d11下禁用或者auto

**Reply by OrangeYoon** [reply]
![image](https://github.com/hooke007/MPV_lazy/assets/169504647/1417ea44-1ae5-42b3-a338-11ac4de8f4c0)
显卡设置的是10位输出，如果关闭是不是就用显示器自己去抖。
如果在d3d11开启auto，是不是会检测到10bit，然后就mpv抖动，把八位变成10位输出？

**Reply by hooke007** [reply]
手册有写，gpu-next下auto对sdr也不会去抖动。

所有窗口内容最终输出都会被统一到显卡设置的位深，中间也可以是软件驱动实现的

你的显示器本身就是8抖10，显卡驱动设置10位自然时时刻刻都在抖动输出。

如果看不出区别，那就没有必要纠结这个选项该设置什么，默认即可
设置10还是强制8还是关闭都是因为遇到了实际的问题才去改动，自行见mpv的相关issue
